claim/gql_mutations.py

```diff
@@ -290,8 +290,11 @@ def update_or_create_claim(data, user):
         [setattr(claim, key, data[key]) for key in data]
     else:
         claim = Claim.objects.create(**data)
+    from core.utils import TimeUtils
     claimed = 0
+    claim.items.update(validity_to=TimeUtils.now())
```

Comment:
@edarchis  is there a service to delete in bulk ?

Reply:
First, the related claim item & services will have soft delete and item/services which aren't meant to delete will be set to null in utils.py "elt.validity_to = None"

Reply:
@delcroip , I will improve in the next PR, thank you.